src/main/resources/default/templates/system/console.html.pasta

```diff
@@ -59,10 +59,14 @@
                 overflow: hidden;
             }
 
-            #main-container {
+            #wrapper-body > #main-container {
                 max-height: 100%;
             }
 
+            #wrapper-body > #page-header, #wrapper-body > #main-container > .row, #wrapper-body > #main-container > .row > #message-box {
```

Comment:
We _could_also overwrite the messages block like `<i:block name="messages"></i:block>` This would disable
the generation of the messages all together instead of creating and the hiding them...